jerry-core/parser/js/lexer.cpp

```diff
@@ -1218,10 +1218,15 @@ lexer_parse_comment (void)
  *      Currently, lexer token doesn't fully correspond to Token, defined in ECMA-262, v5, 7.5.
  *      For example, there is no new-line token type in the token definition of ECMA-262 v5.
  *
+ * Note:
+ *      For Lexer alone, it is hard to find out a token is whether a regexp or a division.
+ *      Parser must set maybe_regexp to true if a regexp is expected.
+ *      Otherwise, a division is expected.
+ *
  * @return constructed token
  */
 static token
-lexer_parse_token (void)
+lexer_parse_token (bool maybe_regexp = false) /**< read '/' as regexp? */
```

Comment:
Default argument value is not required here. Every call site of `lexer_parse_token` passes an argument.